contributor-guide/modules/ROOT/pages/index.adoc

```diff
@@ -12,5 +12,5 @@ Start by reading about xref:getting-involved.adoc[how to get involved in Boost].
 
 == See Also
 
-* https://docs.cppalliance.org/user-guide/index.html[User Guide]
-* https://docs.cppalliance.org/formal-reviews/index.html[Formal Reviews]
+* link:../user-guide/index.html[User Guide]
```

Comment:
We need to fix that with proper xrefs. Relative links going to parent paths might work but they can break very easily.

Reply:
@alandefreitas I have tried many xref options and none have worked. Might need to get some support from somewhere to work out the correct link format.

Reply:
I just tried `xref:user-guide:ROOT:index.adoc[]` and it worked fine.

![image](https://github.com/cppalliance/site-docs/assets/5369819/c9a431e8-e746-46bb-95f4-dd88b9a15041)

So we have the syntax `xref:version@component:module:file.adoc#fragment[text]`, where:

- `version` is empty and therefore implies we want the same version (this playbook has no versions)
- `component` is `user-guide`
- `module` is `ROOT` and it can't be implied because we defined the component and the `file` component can't be ambiguous
- `file.adoc` is `index.adoc`
- `fragment` is empty as usual because we want to refer to the whole page
- `text` is empty because we want it to come from the `adoc` attributes

Using xref 1) lets us get the title from the adoc attributes, 2) doesn't rely on circumstantial patterns in the build layout, 3) allows the asciidoc plugin to perform static analysis (including indicating errors, auto-completion, syntax-highlighting, etc), and 4) removes ".." segments, which are unsafe.

The complete syntax is in https://docs.antora.org/antora/latest/page/xref/

Reply:
works, thanks, done.